app/lib/methods/subscriptions/rooms.js

```diff
@@ -140,6 +140,8 @@ const createOrUpdateSubscription = async(subscription, room) => {
 
 			if (tmp.lastMessage) {
 				const lastMessage = buildMessage(tmp.lastMessage);
+				// This last Message have a different format when merge two streams results in a error caused by the urls object
```

Comment:
The error is relative to the server, this update the last message with a message before parse the urls with OEmbed.